tests/qa/run

```diff
@@ -89,10 +90,12 @@ run_test() {
 }
 
 rc=0
+# By default we use RS_VERSIONS, but with variants they may need to specify their own redis verisons, hence this file
+VERSIONFILE=${HERE}/RS_VERSIONS${VARIANT}
```

Comment:
We send `VARIANT=lite` from `circleci.yaml` but the file name is `RS_VERSIONS-lite` (with a dash) - is that a problem?

Reply:
CircleCI passes in VARIANT=-lite, which we use to build the file. This was validated with @tomerhekredis